gravitee-am-repository/gravitee-am-repository-jdbc-api/src/main/java/io/gravitee/am/repository/jdbc/provider/impl/ConnectionFactoryProvider.java

```diff
@@ -70,6 +70,9 @@ public class ConnectionFactoryProvider {
     public static final long DEFAULT_SETTINGS_MAX_ACQUIRE_TIME = 3000;
     public static final long DEFAULT_SETTINGS_MAX_CREATE_CNX_TIME = 5000;
     public static final String DRIVER_SQLSERVER = "sqlserver";
+    public static final List<String> DRIVERS_SUPPORTING_SCHEMA = List.of("postgresql", "mariadb");
```

Comment:
question: here you have 2 db but in the schemaform.json, you say it is only for psql, which one is valid ?